install_test.sh

```diff
@@ -1,17 +1,42 @@
-#!/bin/sh
```

Comment:
I think this file should be reverted to its original state

Reply:
All of it? Or should I just remove the different shells? What about the part where I added the osx CI?